### DISCUSSION TITLE
Refresh doesn't work when data changed from outside

### DISCUSSION THREAD

**Discussion by zreptil** [pawelsalawa/sqlitestudio#4923] [General]
### Details
When data is changed by another program in the table i am currently looking at in sqlitestudio, i cannot see the changes in sqlitestudio. Refreh doesn't work, and even disconnecting and connecting to the database changes nothing. Exiting sqlitestudio and running it again shows the changed data.

### Steps to reproduce
- open sqlite file in sqlitestudio
- change some data using external tools (php-script or something else)
- hit refresh in the table data view - data doesn't change
- hit disconnect
- hit connect - still no changes in data
- exit sqlitestudio
- start sqlitestudio - data changes are available

### Operating system
Windows 11

### SQLiteStudio version
v3.4.4

**Comment by pawelsalawa** [comment]
If SQLiteStudio doesn't see changes made in external tool, it's not fault of SQLiteStudio. You probably did not commit transaction on that other tool.

**Reply by zreptil** [reply]
It is absolutely correct, that SQLiteStudio may not recognize the change. But I would like to refresh the table manually without having to exit SQLiteStudio. The data was committed and the data-changes can be seen by SQLiteStudio also, when i exit it and start it again. But I cannot make SQLiteStudio reload the file without exiting it. I would think, that disconnecting the database and connecting it again would reload the file from scratch. But this seems not to be the case.
